build.gradle

```diff
@@ -276,8 +276,8 @@ task checkMavenCoordianteCollisions {
         def coordinate = it.publishing?.publications[0].coordinates
         if (coordinates.containsKey(coordinate)) {
           throw new GradleException("Duplicate maven coordinates detected, ${coordinate} is used by " +
-              "both ${coordinates[coordinate]} and ${it.path}.\n" +
```

Comment:
`applySpotless` made these changes, do we want these? ...is my spotless broken?

Reply:
Yes.  Because it's going to keep making them.

Reply:
Well, consistency is king 👍